client/store-management-links/quick-link/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from '@wordpress/element';
+import { external, Icon } from '@wordpress/icons';
+import { __experimentalText as Text } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const QuickLink = ( { icon, title, href, isExternal, onClick } ) => {
+	return (
+		<div className="woocommerce-quick-links__item">
+			<a
+				onClick={ ( e ) => onClick( e ) }
```

Comment:
Could we just use `onClick` here instead of defining a closure?

Reply:
:D yep good spotting

Reply:
This is resolved now when passing to the `Link` component in https://github.com/woocommerce/woocommerce-admin/pull/5476/commits/24e2171719d231dcd39305af5ff443a891f509ea